src/Commands/LaravelRouteTruncateCommand.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Bilfeldt\LaravelRouteStatistics\Commands;
+
+use Illuminate\Console\Command;
+use TypeError;
+
+class LaravelRouteTruncateCommand extends Command
+{
+    public $signature = 'route:truncate-stats';
+
+    public $description = 'Prune route usage statistics';
+
+    public function handle(): int
+    {
+        try {
+            call_user_func_array([config('route-statistics.model'), 'truncate'], []);
```

Comment:
I have intentionally left our a decicated command for this is already possible by the main `model:prune` command:

```
Schedule::command('model:prune', [
    '--model' => [config('route-statistics.model')],
])->daily();
```

What is this command bringing to the table?

Reply:
This command makes it possible to completely empty the log from the console (if preferred).

Reply:
Okay. Feel free to submit this as a separate PR. I am not sure at this point in time if I think this is really a usecase that warrants a dedicated artisan command.

There are also foreign keys that one should perhaps think about. Not really sure what is the best approach here - note there are some packages already doing this: https://en.laracraft.tech/blog/laravel-db-truncate-command

Nevertheless let us separate it from this PR at hand. It has nothing to do with the parameters.